### ISSUE TITLE
:LLPStartPreview doesn't open anything

### ISSUE THREAD

**Issue by addcninblue** [xuhdev/vim-latex-live-preview#70]
## Expected behavior

LLP starts preview

## Actual behavior

My Vim installation recognize the command

`:LLPStartPreview`

but it does nothing.

## Steps to reproduce

Relevant parts of my .vimrc:

```vim
packadd vim-latex-live-preview
autocmd Filetype tex setl updatetime=1

let g:livepreview_previewer = 'mupdf'
```

## System configuration

- **Vim version**:

```bash
NVIM v0.3.0
Build type: Release
LuaJIT 2.0.5
Compilation: /usr/bin/cc -march=x86-64 -mtune=generic -O2 -pipe -fstack-protector-strong -fno-plt -Wconversion -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -O2 -DNDEBUG -DMIN_LOG_LEVEL=3 -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99
-Wimplicit-fallthrough -Wvla -fstack-protector-strong -fdiagnostics-color=auto -Wno-array-bounds -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -I/build/neovim/src/build/config -I/build/neovim/src/neovim-0.3.0/src -I/usr/include -I/build/neovim/src/build/src/nvim/auto -I/build/neovim/src/build/include
Compiled by builduser

Features: +acl +iconv +jemalloc +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info

```

- **Platform**:
Arch Linux

Other stuff:

Output of echo b:livepreview_buf_data after running :LLPStartPreview
```
{'root_dir': '/home/addison/stuff/cs70', 'py_exe': 'python', 'tmp_dir': '/tmp/vim-latex-live-preview-N8THwq', 'run_cmd': 'env TEXMFOUTPU
T=/tmp/vim-latex-live-preview-N8THwq/home/addison/stuff/cs70 TEXINPUTS=/tmp/vim-latex-live-preview-N8THwq/home/addison/stuff/cs70:/home/
addison/stuff/cs70: pdflatex -shell-escape -interaction=nonstopmode -output-directory=/tmp/vim-latex-live-preview-N8THwq/home/addison/st
uff/cs70 /tmp/vim-latex-live-preview-N8THwq/home/addison/stuff/cs70/hw00', 'preview_running': 1, 'tmp_src_file': '/tmp/vim-latex-live-pr
eview-N8THwq/home/addison/stuff/cs70/hw00'}
```

File I'm compiling:

```tex
\documentclass{article}

\begin{document}

testing
testing
testing

\end{document}
```

**Comment by NikosAlexandris** [comment]
Are there no error messages (in vim's bottom line)?

I had something similar here, with an error complaining about the bibliography not being generated. I added a `.bib` file and an example `\cite{SomeKey}` and it worked.

**Comment by V-Kh** [comment]
I have similar problems with Fedora. What details do you need?

**Comment by Newexplorer** [comment]
Check what version of python you have installed. I think the plugin doesn't work with python3.

**Comment by V-Kh** [comment]
I use both versions, however, v2 is my system default as it is everywhere else. 
Nevertheless, it shouldn't be an issue since the plugin checks python versions before starting.

**Comment by ecly** [comment]
Experiencing the same problem with NeoVim after updating the Plugin after about 8 months.

**Comment by RedBorg** [comment]
Experiencing same problem also with evince, pdflatex, neovim and arch, fresh plugin install (with vim-plug) with no configs.

**Comment by xuhdev** [comment]
Folks, I really cannot reproduce this issue here. Have you tried with only this plugin enabled? It can be some conflict but I really don't know.

**Comment by dumuzidsafi** [comment]
Same issue, on Manjaro with gvim. Did anyone figure it out? I'll try with just this plugin alone now.

As with other similar issue reports, the document compiles fine via other means.

I had tried it with both vim-plug and pathogen, but as I've been having problems with both (and yay with AUR packages*), I decided just to plop it into .vim/plugin/, 

The relevant errors are: 
```
Initial compile...
Error detected while processing function UpdatingToggle[2]..LaunchMuPDF:
line 18:
Compilation failed; the PDF does not exist.
Live updating preview now ON.

```
And :LLPStartPreview:
`
Failed to compile
`
And:
`
cat: home/tunurit/.config/live-latex-preview/lastresult: No such file or directory
`
With a pdf compiled via other means, I just get:
```
PDF preview now on.
Live updating preview now ON.

```
with the same lastresult file missing.
